src/model/encoding/convertFromHTMLToContentBlocks.js

```diff
@@ -366,12 +379,23 @@ function genFragment(
   }
 
   var entityId: ?string = null;
-  var href: ?string = null;
 
   while (child) {
     if (nodeName === 'a' && child.href && hasValidLinkText(child)) {
-      href = new URI(child.href).toString();
-      entityId = DraftEntity.create('LINK', 'MUTABLE', {url: href});
+      // had to cast to Object due to https://github.com/facebook/flow/issues/1730
+      const anchor:Object = child;
+
+      var entityConfig = {};
+
+      anchorAttr.forEach(attr => {
+        if (anchor[attr]) {
+          entityConfig[attr] = anchor[attr];
+        }
+      });
+
+      entityConfig.url = new URI(entityConfig.href).toString();
```

Comment:
We probably need to be certain that there is an `href` value here -- the URI module might shout if this is an empty string or something.